app/backend/src/couchers/tasks.py

```diff
@@ -131,24 +131,43 @@ def send_password_changed_email(user):
 
 def send_email_changed_notification_email(user):
     """
-    Send the user an email saying their email has changed. Goes to the old address
+    Send an email to user's original address notifying that it has been changed
     """
     logger.info(
         f"Sending email changed (notification) email to {user=} (old email: {user.email=}, new email: {user.new_email=})"
     )
     email.enqueue_email_from_template(user.email, "email_changed_notification", template_args={"user": user})
 
 
-def send_email_changed_confirmation_email(user, token, expiry_text):
+def send_email_changed_confirmation_to_old_email(user, token, expiry_text):
     """
-    Send the user an email confirming their new email. Goes to the new address
+    Send an email to user's original email address requesting confirmation of email change
     """
     logger.info(
-        f"Sending email changed (confirmation) email to {user=} (old email: {user.email=}, new email: {user.new_email=})"
+        f"Sending email changed (confirmation) email to {user=}'s old email address, (old email: {user.email}, new email: {user.new_email=})"
     )
+
+    confirmation_link = urls.change_email_link(confirmation_token=token)
```

Comment:
You use the same `change_email_link` for both new and old emails, and frontend won't know which of the two functions to call. I suggest you combine them into one and return as in other comment